games/m.yaml

```diff
@@ -347,6 +347,26 @@
   updated: 2016-03-16
   url: https://www.meridiannext.com
 
+- name: Metaforce
+  lang:
+  - C++
+  - Python
+  - C
+  license:
+  - MIT
+  development: active
+  originals:
+  - Metroid Prime
+  repo: 'https://github.com/AxioDL/metaforce'
+  type: remake
+  url: 'https://axiodl.com/'
+  status: playable
+  info: engine only
+  updated: '2022-03-01'
+  images:
+  - https://github.com/AxioDL/metaforce/blob/main/assets/metaforce-screen1.png?raw=true
```

Comment:
```suggestion
  - https://github.com/AxioDL/metaforce/blob/main/assets/metaforce-screen1.png?raw=true
  - https://axiodl.com/images/hecl_hu25398d5c0d2da88c4a5221db09696c79_1724775_0x480_resize_bg292a2d_box_2.png
  - https://axiodl.com/images/pwe_hu7c7bec678fa5eef23b41aaa434dfa04c_1742668_0x480_resize_bg292a2d_box_2.png
  video:
    youtube: uMwqpL2_-LY
```

Reply:
The 2 extra images are actually Prime World Editor, not necessarily Metaforce

Reply:
I will include them though since they are related projects